connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -207,8 +185,30 @@ public void close() {
         backup.stop();
     }
 
+
+
     @Test
     public void testReplication() throws InterruptedException {
+
+        // create consumers before starting the connectors so we don't need to wait for discovery
+        Consumer<byte[], byte[]> consumer3 = primary.kafka().createConsumerAndSubscribeTo(Collections.singletonMap(
```

Comment:
It's a bit unusual to have `consumer3` and `consumer4` without 1 and 2 =)

Reply:
updated to `consumer1` and `consumer2`